app/assets/stylesheets/release.scss

```diff
@@ -1,11 +1,22 @@
 @import "settings";
 
+.genres {
+  z-index: 2;
+}
+
+.slider {
+  z-index: 1;
+}
+
 .release-form {
   background-color: $white;
   border-radius: 10px;
   color: $black;
   padding: 2em;
-
+  
```

Comment:
added is white space because hound first said that my declaration of label in the next line should have a whitespace in between.